src/commands/CommandEditAlertParserForm.js

```diff
@@ -0,0 +1,164 @@
+                'userGroup[name]',
+            ],
+            paging: 'false',
+        },
+    },
+}
+
+const mutation = {
+    resource: 'smsCommands',
+    type: 'update',
+    partial: true,
+    id: ({ id }) => id,
+    data: ({ command }) => command,
+}
+
+export const CommandEditAlertParserForm = ({ id }) => {
+    const history = useHistory()
+    const engine = useDataEngine()
+    const updateCommand = command =>
+        engine
+            .mutate(mutation, { variables: { command, id } })
+            .then(() => {
+                history.push(SMS_COMMAND_LIST_PATH)
+            })
+            .catch(error => {
+                const isValidationError = error.type === 'access'
+
+                // Potential validation error, return it in a format final-form can handle
+                if (isValidationError) {
+                    const fallback = 'No error message was provided'
+                    const message = error.message || i18n.t(fallback)
+
+                    return {
+                        [FORM_ERROR]: message,
+                    }
+                }
+
+                // Throw any unexpected errors
+                throw error
+            })
+
+    const { loading, error, data } = useDataQuery(query, { variables: { id } })
+
+    if (loading) {
+        return (
+            <CenteredContent>
+                <CircularLoader />
+            </CenteredContent>
+        )
+    }
+
+    if (error) {
+        const msg = i18n.t(
+            'Something went wrong whilst loading the sms command'
+        )
+
+        return (
+            <NoticeBox error title={msg}>
+                {error.message}
+            </NoticeBox>
+        )
+    }
+
+    const { name, parserType, receivedMessage, userGroup } = data.smsCommand
+    const initialValues = {
+        name,
+        parserType,
+        receivedMessage,
+    }
+
+    // The user group display is just a dummy field
+    return (
+        <Form onSubmit={updateCommand} initialValues={initialValues}>
+            {({
+                handleSubmit,
+                submitting,
+                pristine,
+                submitError,
+                hasSubmitErrors,
+            }) => (
+                <form onSubmit={handleSubmit}>
+                    <FormRow>
+                        <FieldCommandName />
+                    </FormRow>
+                    <FormRow>
+                        <FieldCommandParser disabled />
+                    </FormRow>
+                    <FormRow>
+                        <SingleSelectField
+                            selected={userGroup.name}
+                            label={i18n.t('User Group')}
+                            disabled
+                        >
+                            <SingleSelectOption
+                                label={userGroup.name}
+                                value={userGroup.name}
+                            />
+                        </SingleSelectField>
```

Comment:
is this really a select with one option? Why do we need that?

Reply:
Yeah it's the field that the comment up top is referring to. It's a dummy field that just shows the current value. I just took it from the old app, but maybe I'm misinterpreting the purpose. To me it seemed like it was just there to show the current value

Reply:
<img width="517" alt="Screenshot 2020-08-26 at 17 22 54" src="https://user-images.githubusercontent.com/7355199/91323114-ce5a8f80-e7c0-11ea-808b-7a33122dcfd1.png">

Reply:
<img width="967" alt="Screenshot 2020-08-26 at 17 23 26" src="https://user-images.githubusercontent.com/7355199/91323159-e03c3280-e7c0-11ea-954f-3fe6016706f1.png">

Reply:
I'm doing the same, but I would probably use the existing `FieldUserGroup`, then you have it in your form state.
It seems like not providing the value is still working, but I'd prefer to be explicit here.

Not that important for now though, I'll align all edit forms after the release anyways

Reply:
👍  thanks for clearing that up!

Reply:
> I'm doing the same, but I would probably use the existing FieldUserGroup, then you have it in your form state.
It seems like not providing the value is still working, but I'd prefer to be explicit here.
>
> Not that important for now though, I'll align all edit forms after the release anyways

Ah I missed that that already existed. I'll use that one instead then 👍

Reply:
Ok, should be good now.